src/com/ableton/VirtualEnv.groovy

```diff
@@ -70,6 +70,14 @@ class VirtualEnv implements Serializable {
     this.venvBinDir = "${venvRootDir}/${activateSubDir}"
   }
 
+  @NonCPS
+  static final String posixPath(String path) {
```

Comment:
Does it really need to be annotated with `@NonCPS`?

Reply:
Probably not, but why not? It is non-CPS-compatible, and we already need to import that symbol for `randomName`.

Reply:
I guess you misunderstood my question.
```suggestion
  static final String posixPath(String path) {
```

Reply:
No, I understood your question correctly. As the function does not use any pipeline steps, it should ideally by annotated with `@NonCPS` in order to optimize Jenkins' performance.